rudder-server/SOURCES/rudder-server-postinst

```diff
@@ -47,9 +47,9 @@ then
       echo "INFO: External database detected, skipping database creation"
       CREATE_DB="false"
     else
-      USER=$(grep rudder.jdbc.username /opt/rudder/etc/rudder-web.properties)
-      CHK_PG_USER=$(su postgres -c "psql -t -c \"select count(1) from pg_user where usename = '${USERNAME}'\"")
-      if [ ${CHK_PG_DB} -ne 0 ]
+      USER=$(grep rudder.jdbc.username /opt/rudder/etc/rudder-web.properties | cut -d'=' -f2)
```

Comment:
This grepped the whole line which make an invalid user name 
 
grep rudder.jdbc.username /opt/rudder/etc/rudder-web.properties 
rudder.jdbc.username=rudder